packages/next/lib/eslint/writeDefaultConfig.ts

```diff
@@ -70,14 +70,14 @@ export async function writeDefaultConfig(
     }
   } else {
     await fs.writeFile(
-      '.eslintrc',
+      '.eslintrc.json',
```

Comment:
This should probably be `.js` because it has comments, right?

Reply:
Oh I see eslint supports comments in JSON https://eslint.org/docs/user-guide/configuring/configuration-files#comments-in-configuration-files